flutter/lib/src/web/html_sentry_js_binding.dart

```diff
@@ -13,10 +13,26 @@ class HtmlSentryJsBinding implements SentryJsBinding {
 
   @override
   void init(Map<String, dynamic> options) {
+    if (options['defaultIntegrations'] != null) {
+      options['defaultIntegrations'] = options['defaultIntegrations']
+          .map((String integration) => _createIntegration(integration));
+    }
+
     _sentry ??= context['Sentry'] as JsObject;
     _sentry!.callMethod('init', [JsObject.jsify(options)]);
   }
 
+  JsObject? _createIntegration(String integration) {
+    switch (integration) {
+      case SentryJsIntegrationName.globalHandlers:
```

Comment:
Just check the JS SDK, don't we need to provide options to GlobalHandler integration, or is this not relevant here?

https://github.com/getsentry/sentry-javascript/blob/7fb76328ecde32dd92c335af2fca08206502024c/packages/deno/src/integrations/globalhandlers.ts#L11C48-L11C74

Reply:
we don't need to, the options are enabled by default and I don't think we'll ever need to adjust them

Reply:
code sample from the repo:

 ```js
 Sentry.init({
   integrations: [
     Sentry.globalHandlersIntegration(),
   ],
 })
 ```